src/gourmet/GourmetRecipeManager.py

```diff
@@ -200,7 +200,7 @@ def go_menu (self):
         separator.show()
         for rc in list(self.rc.values()):
             i=Gtk.MenuItem("_%s"%rc.current_rec.title)
-            i.connect('activate',rc.show)
```

Comment:
Looking further after I committed, I think this is dead code, and only the change below made a difference.